test/e2e/tests/smart-transactions/mocks.ts

```diff
@@ -262,6 +263,61 @@ const GET_BLOCK_BY_HASH_RESPONSE = {
 };
 
 export async function mockSmartTransactionRequests(mockServer: MockttpServer) {
```

Comment:
I'm seeing this function is also used in other tests `smart-transactions.spec.ts`. do we want to have the below changes inside this existing mock function? or rather in the `mockSmartTransactionBatchRequests` ?
In those tests we are not using mainnet, as selected chain id, so it won't have any effect, but just wondering

Reply:
This is intentional to avoid duplication, but there should be no changes to the existing logic and tests as we're just using a central `mockSmartTransactionRequestsBase` function.